lib/serialize.js

```diff
@@ -115,15 +116,14 @@ module.exports = function (app, defaults) {
       app: app,
       model: model,
       method: ctx.method.name,
+      meta: ctx.meta ? utils.clone(ctx.meta) : null,
```

Comment:
I wasn't super happy about polluting the ctx object with another property, but I didn't see an alternative that would allow us to pass the value around.